css_docs/images.css

```diff
@@ -0,0 +1,52 @@
+@import url('/utility.css');
+@import url('./css_docs/colors.css');
+@import url('./css_docs/Typography.css');
+@import url('./css_docs/Reset.css');
+
+
+
+.img-fluid{
+    width: 500px;
```

Comment:
Give max-width an absolute value. To make responsive images you would need to give width:100% generally for all images